File: src/main/java/ru/zdoher/japs/repositories/KanjiRepository.java
package ru.zdoher.japs.repositories;

import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;
import ru.zdoher.japs.domain.Kanji;
import ru.zdoher.japs.repositories.aggregation.RepositoryAggregation;

@Repository
public interface KanjiRepository
        extends CrudRepository<Kanji, String>, RepositoryAggregation<Kanji> {
}

File: src/main/java/ru/zdoher/japs/repositories/KanjiRepository.kt
package ru.zdoher.japs.repositories

import org.springframework.data.mongodb.repository.ReactiveMongoRepository
import org.springframework.stereotype.Repository
import ru.zdoher.japs.domain.Kanji
import ru.zdoher.japs.repositories.aggregation.RepositoryAggregation

@Repository
interface KanjiRepository : ReactiveMongoRepository<Kanji, String>, RepositoryAggregation<Kanji>